kite-data/kite-data-core/src/main/java/org/kitesdk/data/Datasets.java

```diff
@@ -364,23 +435,25 @@ public static boolean exists(URI uri) {
   }
 
   /**
-   * Check if a {@link Dataset} identified by the given URI string exists.
+   * Check whether a {@link Dataset} identified by the given URI string exists.
    * <p>
-   * URI formats are defined by {@code Dataset} implementations, but must begin
-   * with "dataset:".
+   * URIs must begin with {@code dataset:}. The remainder of
+   * the URI is implementation specfic, depending on the dataset scheme.
    *
-   * @param uri a {@code Dataset} URI string.
-   * @return {@code true} if the dataset exists, {@code false} otherwise
+   * @param uri a {@code Dataset} URI string
+   * @return {@code true} if the dataset exists, or {@code false}
    */
   public static boolean exists(String uri) {
     return exists(URI.create(uri));
   }
 
   /**
-   * List the {@link Dataset} URIs in the repository identified by the URI
+   * List the {@link Dataset} URIs in the repository identified by the URI.
    * <p>
    * URI formats are defined by {@code Dataset} implementations. The repository
-   * URIs passed to this method must begin with "repo:".
+   * URIs you pass to this method must begin with {@code repo:}. For example, to 
+   * list the {@code Dataset} URIs for the Hive repository {@code /default}, 
+   * provide the URI {@code repo:hive:/default}.
```

Comment:
Changed to:

   * URI formats are defined by {@code Dataset} implementations. The repository
   * URIs you pass to this method must begin with {@code repo:}. For example, to 
   * list the {@code Dataset} URIs for the Hive repository, provide the URI
   * {@code repo:hive}.